app/src/main/java/com/example/hungrywolfes/ui/SplashScreenActivity.kt

```diff
@@ -1,24 +1,24 @@
-package com.example.hungrywolfes
+package com.example.hungrywolfes.ui
 
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.os.Handler
-import android.os.Looper
+import androidx.lifecycle.lifecycleScope
+import com.example.hungrywolfes.R
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
 
 class SplashScreenActivity : AppCompatActivity() {
 
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_splash_screen)
 
-        Handler(Looper.getMainLooper()).postDelayed({
-
-            val intent= Intent(this,MainActivity::class.java)
+        lifecycleScope.launch {
+            delay(1000)
```

Comment:
You can extract this on a constant value instead of hardcoding directly. It's a good practice